frontend/src/app/modules/interventions/list/table/table.component.html

```diff
@@ -1,48 +1,52 @@
-<div class="mat-elevation-z8">
-  <mat-table [dataSource]="dataSource">
-    <!-- Position Column -->
-    <ng-container matColumnDef="position">
-      <mat-header-cell *matHeaderCellDef>L.p.</mat-header-cell>
-      <mat-cell *matCellDef="let element" data-label="Numer">{{ element.position }}</mat-cell>
-    </ng-container>
+<eko-loader [loading$]="dataSource.loading$" [cover]="true">
+  <div class="mat-elevation-z8">
+    <mat-table [dataSource]="dataSource" matSort matSortDisableClear matSortActive="date" matSortDirection="desc">
+      <!-- Position Column -->
+      <ng-container matColumnDef="position">
+        <mat-header-cell *matHeaderCellDef>L.p.</mat-header-cell>
+        <mat-cell *matCellDef="let index = index" data-label="Numer">{{
+          index + this.paginator.pageIndex * this.paginator.pageSize + 1
```

Comment:
I didn't know it is possible to use `this` in templates :)

Just a side note, it should also work without 'this' as well - template compiler will browse through public properties of component class when encountering a property name in the template.

Reply:
Heh, working with Angular in an unorthodox way. Anyway, I've removed this.

Reply:
Hmm, it was ok for me, I was just surprised :)